src/js/components/StarRating/StarRating.js

```diff
@@ -3,11 +3,14 @@ import { Star } from 'grommet-icons/icons/Star';
 import { StarOutline } from 'grommet-icons/icons/StarOutline';
 import { FormContext } from '../Form/FormContext';
 import { RadioButtonGroup } from '../RadioButtonGroup';
+import { StyledThumbsRatingBox } from '../ThumbsRating/ThumbsRating';
```

Comment:
can we also make StarRating its own box instead of importing `StyledThumbsRatingBox`
maybe `StyledStarRatingBox` that way incase we have styles that are specific to either component they are mutually exclusive.